src/category_theory/adjunction/limits.lean

```diff
@@ -23,18 +23,34 @@ include adj
 section preservation_colimits
 variables {J : Type v} [small_category J] (K : J ⥤ C)
 
+/--
+The right adjoint of `cocones.functoriality K F : cocone K ⥤ cocone (K ⋙ F)`.
+
+Auxilliary definition for `functoriality_is_left_adjoint`.
+-/
 def functoriality_right_adjoint : cocone (K ⋙ F) ⥤ cocone K :=
 (cocones.functoriality _ G) ⋙
   (cocones.precompose (K.right_unitor.inv ≫ (whisker_left K adj.unit) ≫ (associator _ _ _).inv))
 
 local attribute [reducible] functoriality_right_adjoint
 
+/--
+The unit for the adjunction for of `cocones.functoriality K F : cocone K ⥤ cocone (K ⋙ F)`.
+
+Auxilliary definition for `functoriality_is_left_adjoint`.
```

Comment:
Or maybe "for"? Not sure about the grammar here. Similarly for `functoriality_counit`.
```suggestion
The unit for the adjunction of `cocones.functoriality K F : cocone K ⥤ cocone (K ⋙ F)`.

Auxiliary definition for `functoriality_is_left_adjoint`.
```